app/src/test/java/com/github/sdp/mediato/api/themoviedb/TMDBApiTest.java

```diff
@@ -0,0 +1,137 @@
+package com.github.sdp.mediato.api.themoviedb;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import androidx.annotation.NonNull;
+
+import com.github.sdp.mediato.api.themoviedb.models.TMDBMovie;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.concurrent.CompletableFuture;
+
+import mockwebserver3.Dispatcher;
+import mockwebserver3.MockResponse;
+import mockwebserver3.MockWebServer;
+import mockwebserver3.RecordedRequest;
+
+public class TMDBApiTest {
+    private static final String APIKEY = "apiKey";
+    private static final String SEARCHTERM = "searchTerm";
+    public static MockWebServer mockApi;
```

Comment:
could be final

Reply:
db can't be final since we need to create it again before each test.
mockApi has been changed